base/tools/src/main/java/com/netscape/cmstools/kra/KRAKeyFindCLI.java

```diff
@@ -78,6 +78,10 @@ public void createOptions() {
         option.setArgName("realm");
         options.addOption(option);
 
+        option = new Option(null, "ownerName", true, "ownerName: foouser");
```

Comment:
The `true` here means that the option takes an argument, i.e. `--ownerName <value>`. See https://commons.apache.org/proper/commons-cli/apidocs/org/apache/commons/cli/Option.html#Option-java.lang.String-java.lang.String-boolean-java.lang.String-.